aiosu/v2/client.py

```diff
@@ -122,8 +123,12 @@ def check_token(func: F) -> F:
     @functools.wraps(func)
     async def _check_token(self: Client, *args: Any, **kwargs: Any) -> Any:
         token = await self.get_current_token()
-        if datetime.utcnow() > token.expires_on:
-            await self._refresh()
+        try:
```

Comment:
```suggestion
        if datetime.now(tz=timezone.utc) > token.expires_on:
```

Reply:
if this change is needed (not sure why you were getting typeerror from this), a try except is a poor way to handle it. just pass the timezone info always

Reply:
i assume this is from passing a date object when creating the token. can just switch this function from a naive dt check